selftests/unit/test_utils_partition.py

```diff
@@ -45,12 +37,8 @@ class TestPartition(unittest.TestCase):
 
     @unittest.skipIf(missing_binary('mkfs.ext2'),
                      "mkfs.ext2 is required for these tests to run.")
-    @unittest.skipIf(missing_binary('sudo'),
+    @unittest.skipIf(not process.can_sudo(),
```

Comment:
The thing with `avocado.utils.process.can_sudo()` is that it assumes a sudo configuration in which a user can run *anything* (or, for some interesting reason, `id -u` only).  It's not uncommon to have sudo configurations that allow for specific commands (or sets of commands).

To avoid those false posivites, and be (optionally) more precise, I recommend we make `avocado.utils.process.can_sudo()` accept an optional command parameter.